easypoll/widget/EasyPoll.jsx

```diff
@@ -55,13 +55,15 @@ const tabs = {
   },
 };
 
-const getFirstSBTToken = () => {
-  const view = Near.view("registry.i-am-human.near", "sbt_tokens_by_owner", {
-    account: `${context.accountId}`,
-    issuer: "fractal.i-am-human.near",
+//verify human with nada.bot
+function isHumanWithNadaBot() {
+  Near.asyncView("v1.nadabot.near", "is_human", {
+    account_id: context.accountId,
+  }).then((result) => {
+    console.log(result);
   });
-  return view?.[0]?.[1]?.[0];
-};
+  return result;
```

Comment:
Wouldn't result be undefined here?

Reply:
No, it returns result in boolean

Reply:
Are you sure? result is out of the "then" scope. [Javascript scope](https://www.w3schools.com/js/js_scope.asp)